layout/style/RunCbindgen.py

```diff
@@ -19,13 +19,23 @@ def _get_crate_name(crate_path):
 
 CARGO_LOCK = mozpath.join(buildconfig.topsrcdir, "Cargo.lock")
 
+SAME_CRATE_FLAG = '--use-same-crate'
+
 def generate(output, cbindgen_crate_path, *in_tree_dependencies):
+    in_tree_dependencies = list(in_tree_dependencies)
+
     env = os.environ.copy()
     env['CARGO'] = str(buildconfig.substs['CARGO'])
 
+    if SAME_CRATE_FLAG in in_tree_dependencies:
+        crate_path = cbindgen_crate_path
+        in_tree_dependencies.remove(SAME_CRATE_FLAG)
+    else:
+        crate_path = mozpath.join(buildconfig.topsrcdir, "toolkit", "library", "rust")
```

Comment:
I will forward this part to @heycam .

Reply:
I'll post this file as separate patch for m-c.

Reply:
posted in https://bugzilla.mozilla.org/show_bug.cgi?id=1616161
I'll merge this once it gets landed